recipes/cppcheck/all/conandata.yml

```diff
@@ -1,22 +1,7 @@
 sources:
-  "2.7.5":
```

Comment:
a lot of the comments are hidden - but is there any reason to remove these versions?

Reply:
It's not a lib where you need a specific version, its a tool and from my perspective there is no need to keep 1.5 y old versions

Reply:
Thats a fair point of view however more broadly that's not always true. https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/reviewing.md#removing-old-versions.

Please refer to our documented practices -- if you think they could be improved don't hesitate to submit a PR.

Unless there is a technical reason they should be kept :) It's beneficial to others

Reply:
Is it possible to set the use_z3 to deprecated for 2.8 and 2.9 and to true for 2.7? Or would that be a technical reason?

Reply:
You should invalidate the use of this option with such logic: https://github.com/conan-io/conan-center-index/blob/master/recipes/sentry-native/all/conanfile.py#L132-L134

Reply:
I restored version 2.7.5 and removed option use_z3